python/lsst/meas/extensions/ngmix/LMSimpleShape.py

```diff
@@ -150,7 +151,8 @@ def measure(self, measRecord, exposure):
             shape = psfArray.shape
             image = lsst.afw.image.ImageD(shape[1], shape[0])
             evaluate = measRecord.get(self.psfMsfKey).evaluate()
-            evaluate.addToImage(image.getArray(), lsst.afw.geom.Point2I(-(shape[0] - 1)//2, -(shape[1] - 1)//2))
+            evaluate.addToImage(
+                image.getArray(), lsst.afw.geom.Point2I(-(shape[0] - 1)//2, -(shape[1] - 1)//2))
```

Comment:
this change looks weird. Move the first argument up a line and just have the second argument on the new line with proper indenting.